kubernetes-pipeline/README.md

```diff
@@ -0,0 +1,192 @@
+# WSO2 Kubernetes Pipeline Helm Chart
+
+## Chart Details
+
+WSO2 Kubernetes Pipeline provides tools and a preconfigured pipeline used for continuous integration and deployment.
+The setup is deployed on top of Kubernetes using Helm, which makes the processes of configuring, installing, scaling and upgrading, quite simple.
+
+Following are the tools used to install and monitor the pipeline
+
+- Jenkins: For continuous integration
+- Spinnaker: For continuous deployment to Kubernetes
+- ELK: For centralized logging
+- Prometheus Operator: For monitoring deployments and visualization using Grafana
+
+![Architecture Diagram](pipeline_architecture.jpg)
+
+## Installation
+
+### Prerequisites
+
+- Install and set up Helm.
+
+- Install NGINX Ingress Controller Git release tag `nginx-0.22.0`.
```

Comment:
@Aaquiff is the specific release tag version `nginx-0.22.0` a prerequisite, anymore?

Reply:
Removed reference to Git release tag